code/modules/antagonists/incursion/incursion.dm

```diff
@@ -91,9 +91,18 @@
 	log_admin("[key_name(admin)] made [key_name(new_owner)] and [key_name(new_owner.current)] into incursion traitor team.")
 
 /datum/antagonist/incursion/proc/equip(var/silent = FALSE)
-	var/obj/item/uplink/incursion/uplink = new(owner, owner.key, 15)
-	owner.current.equip_to_slot(uplink, ITEM_SLOT_BACKPACK)
-	to_chat(owner.current, "<span class='notice'><b>You have been equipped with a syndicate uplink located in your backpack. Activate the transponder in hand to access the market.</b></span>")
+	var/obj/item/uplink/incursion/uplink = new(owner.current.loc, owner.key, 15)
```

Comment:
maybe doing it at the turf is better in case the owner is inside of something (somehow) such as a vehicle.

Reply:
Isn't .loc their turf?

Reply:
Loc is the object they are inside (Which is a turf if they are on a turf.)
If they are in a locker then it will spawn in the locker which is fine.
The **only** case this will ever effect is if for some reason someone is inside something which items cannot be removed from when they become an incursionist AND they have no backpack and no slots to go to, but might as well handle it anyway just in case.